### PR TITLE
[IMP] outlook mail_plugin: add solution for cookie issue

### PR DESCRIPTION
**Pull request by lse-odoo**

### CODE REVIEW

**Comment by lse-odoo** [comment]
@Feyensv could you review it please ? :) It's been a loooon while that I didn't made a PR for the documentation, let me know if I need to ping someone in particular for that

**Comment by AntoineVDV** [comment]
There was an issue with the code owners mapping. It's not fixed :rocket:

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (meng-odoo) [inline content/applications/productivity/mail_plugins/outlook.rst:79]: I made some small changes to this sentence to improve sentence flow.

`If, after a short while, the panel is still empty, it is possible that your browser cookie`

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (meng-odoo) [inline content/applications/productivity/mail_plugins/outlook.rst:80]: Small changes to improve sentence flow.

`settings prevented it from loading.`

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (meng-odoo) [inline content/applications/productivity/mail_plugins/outlook.rst:81]: Small changes to improve sentence flow.

`Note that these settings also change if you are in "Incognito" mode on your`

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (meng-odoo) [inline content/applications/productivity/mail_plugins/outlook.rst:86]: Small changes to improve wording and punctuation.

`For Google Chrome, you can do so by following the guide at`

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (meng-odoo) [inline content/applications/productivity/mail_plugins/outlook.rst:84]: Deleted "you can" so that it reads more like instructions, rather than an option.

`To fix this issue, configure your browser to always allow cookies on Odoo's plugin page.`

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (meng-odoo) [inline content/applications/productivity/mail_plugins/outlook.rst:88]: Small changes to keep the grammar consistent throughout the sentence.

`and adding download.odoo.com to the list of Sites that can always use cookies.`

Please note that I removed the backtick punctuation around "download.odoo.com" and "Sites that can always use cookies" in the corrected quote, because Github reads it as formatting.

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (meng-odoo) [inline content/applications/productivity/mail_plugins/outlook.rst:94]: Fixed a small grammar mistake.

`Once done, the panel needs to be opened again.`

Also, which panel are you referring to? It would be good to specify.

**Review by meng-odoo — CHANGES_REQUESTED** [review]
Hello! I'm requesting some small changes before we approve. Thanks :)

**Comment by lse-odoo** [comment]
@meng-odoo thank you for your review. I did update the text accordingly (in theory). Let me know your thoughts on the matter

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (meng-odoo) [inline content/applications/productivity/mail_plugins/outlook.rst:92]: Let's delete this image. For platforms outside of Odoo, we want to link to guides from those platforms, rather than walking through the instructions on Odoo's documentation. So what you did here by linking to Google's support page is perfect, and we don't need to include more Google Chrome content in addition to that.

Also, we try not to use too many images. Although your image is helpful, it is not completely necessary for the reader to understand the instructions.

**Review by meng-odoo — CHANGES_REQUESTED** [review]
Hi! Thank you very much for making my suggested changes. All of the changes look good.

Upon a second review, I believe we should change a couple more key things before approving: making it a Tip instead of a Warning, and removing the image. Please see my comments below and let me know if you have any questions :)

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (AntoineVDV) [reply content/applications/productivity/mail_plugins/outlook.rst:78]: @meng-odoo Speaking about tips, you can click on the "Add a suggestion button" or use `ctrl-g` to easily make suggestions.

For example:
```suggestion
   .. tip::
```

cc @StraubCreative

**Comment by lse-odoo** [comment]
@meng-odoo I do think that adding pictures can help customers visualise what to do and if the settings did change with time. But I do understand that it can indeed be confusing for some. I did remove it as you asked (and changed to a tip).
Let me know if you notice something else :)

**Review by meng-odoo — APPROVED** [review]
@lse-odoo Hi, thanks again for making the suggested changes! Everything looks great. I'll go ahead and move this to the next stage in the review process.

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (AntoineVDV) [inline content/applications/productivity/mail_plugins/outlook.rst:82]: Nitpicking for your future PRs: It's better not to return to a new line if you don't intend to start a new paragraph. Indeed, single line breaks have no effect on the rendered HTML so it can be confusing for the technical reader to see useless line breaks.

File: content/applications/productivity/mail_plugins/outlook.rst
Comment (AntoineVDV) [inline content/applications/productivity/mail_plugins/outlook.rst:87]: Nitpicking for your future PRs: [you can break the line anywhere inside a link label](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character).
```suggestion
      `https://support.google.com/chrome/answer/95647
<https://support.google.com/chrome/answer/95647#:~:text=Allow%20or%20block%20cookies%20for%20a%20specific%20site>`_
```

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**
